javascript/javascript1/week4/script2.js

```diff
@@ -0,0 +1,46 @@
+//Vowel Count
+
+function vowelCount(letters) {
+  let count = 0;
+  for (let i = 0; i < letters.length; i++) {
+    let letter = letters.charAt(i).toLocaleLowerCase();
+    if (
+      letter === "a" ||
+      letter === "e" ||
+      letter === "i" ||
+      letter === "o" ||
+      letter === "u"
+    ) {
```

Comment:
This approach is okay. But try to use an array to store the vowels and check if the letter has digit similar to array. Makes the code cleaner and simpler and reusable